openedx/features/idea/models.py

```diff
@@ -2,5 +2,77 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import FileExtensionValidator
+from django_countries.fields import CountryField
 
-# Create your models here.
+from functools import partial
+
+from lms.djangoapps.onboarding.models import Organization
+from .helpers import upload_to_path
+
+
+class Location(models.Model):
+    country = CountryField()
+    city = models.CharField(max_length=255)
+
+    @property
+    def location(self):
+        return '{city}, {country}'.format(city=self.city, country=self.country)
+
+    def __unicode__(self):
```

Comment:
Same, how will it be used?

Reply:
We've multiple unicodes all will be inherited in Idea, which one will work?

Reply:
It will be used if any model inherit this abstract model and do not provide `__unicode__`, Idea model has its own '__unicode__'

Reply:
But do you think it's correct behavior in case some Model doesn't provide __unicode__. Let's say class Customer implemented this but didn't provide __unicode__. When ever Customer object's unicode method will be used it will return customer address.